docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc

```diff
@@ -196,7 +194,7 @@ You can monitor the SQ shell log file : /home/trafodion/apache-trafodion-1.3.0-i
 Startup time  0 hour(s) 1 minute(s) 9 second(s)
 Apache Trafodion Conversational Interface 1.3.0
 Copyright (c) 2015 Apache Software Foundation
->> initialize trafodion;
+>> 
```

Comment:
The command is piped into sqlci so it's not shown. It's very weird but I'm
documenting behavior, not changing it.

On Thu, Feb 25, 2016 at 5:10 PM, DaveBirdsall <notifications@github.com>
wrote:

> In docs/provisioning_guide/src/asciidoc/_chapters/script_install.adoc
> <https://github.com/apache/incubator-trafodion/pull/341#discussion_r54186985>
> :
>
> > @@ -196,7 +194,7 @@ You can monitor the SQ shell log file : /home/trafodion/apache-trafodion-1.3.0-i
> >  Startup time  0 hour(s) 1 minute(s) 9 second(s)
> >  Apache Trafodion Conversational Interface 1.3.0
> >  Copyright (c) 2015 Apache Software Foundation
> > ->> initialize trafodion;
> > +>>
>
> Looks like we have a mystery command now that sqlci completed.
>
> I'm guessing you wanted to delete this prompt and also the "SQL opteration
> complete" message after it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/341/files#r54186985>.
>



-- 
Thanks,

Gunnar
*If you think you can you can, if you think you can't you're right.*